experimental/Schemes/duValSing.jl

```diff
@@ -0,0 +1,225 @@
+function has_du_val_singularities(F::MPolyDecRingElem)::Bool
```

Comment:
Easily!
But before implementing that, we would have to agree (among all classifiers, not just this basic one) on an output format for singularity types so that we can output tuples consisting of the point and the singularity.

Reply:
For DuVal singularities I suggest `((0,0), (:A, 4))`
The format `(:A, n)`  `(:D ,n)`, `(:E ,n)` is what we use for the root lattices.

Reply:
Great. Thanks. I will do this for the hypersurface case -- for the higher codimension case, this will come as soon as a 'vdim' (by whatever name) is available for modules.

Reply:
The more fancy classifiers return normal forms with all kinds of additional data associated to the singularity. That would be overkill here. So the format is fine.